views/showcase.jade

```diff
@@ -0,0 +1,18 @@
+extends layout
+
+block content
+    h2 Showcase
+    - each item in config.showcase
```

Comment:
To match style/formatting of the reset of the site, this should match bootswatch (see: https://github.com/MaxCDN/bootstrap-cdn/blob/develop/views/bootswatch.jade). You need wrap everything after the `h2 Showcase` in a `.well.text-center`, but can replace the `row` with the `offset` element I mentioned below.